client/src/components/Visualiser/Controller/Control.module.scss

```diff
@@ -64,6 +64,10 @@
   width: 40px;
 }
 
+.timelineSlider {
+    width: 100%;
+}
+
 // .progressBar {
```

Comment:
Do we still need this?

Reply:
Since we are using an html input element for our progress slider yeah since i wanted the slider to take up the full width of the div